controllers/flinkcluster_updater.go

```diff
@@ -734,7 +735,11 @@ func (updater *ClusterStatusUpdater) getJobStatus() *v1beta1.JobStatus {
 			var jobSubmitInProgress = observedJob != nil &&
 				observedJob.Status.Succeeded == 0 &&
 				observedJob.Status.Failed == 0
-			if jobSubmitSucceeded || jobSubmitInProgress {
+			// Flink job submit pod is running
```

Comment:
adjust comment, such like:

We found there is a strange situation: Job Completed but Pod is still Running.
So we add logic below to avoid this issue.